src/pages/error.vue

```diff
@@ -1,28 +1,29 @@
 <template>
-    <div class="background uk-background-primary">
-
-        <div>
-            <img src="core/gfx/cloud-logo-invert.svg" width="50%">
-        </div>
-        <div>
-            <div class="uk-position-center uk-background-muted uk-border-rounded uk-padding">
-                <h2 v-translate>
-                    Could not Authenticate
-                </h2>
-                <div v-translate class="uk-text-small">
-                    Please click the button below again to authenticate with ownCloud and get access to your data.
-                </div>
-                <div class="uk-padding">
-                    <button class="uk-button uk-button-primary" @click="authenticate" v-translate>Authenticate</button>
-                </div>
-            </div>
-        </div>
-    </div>
+  <div>
+    <v-layout class="background primary">
+      <v-img
+      v-if="configuration.theme.logo.big"
+      :src="configuration.theme.logo.big"
+      width="50%"
+      :aspect-ratio="1.96">
+      <v-flex class="theme--light grey lighten-2 pa-5 ma-3 center-dialog" xs4>
```

Comment:
"pa-5" and "ma-3" are these hard-coded values ?

what if we want to change the theme to have a different padding for these kind of components ?
might make more sense to have something like "pa-small", "pa-medium", "pa-large"

Reply:
https://vuetifyjs.com/en/layout/spacing - this are the spacing helpers - they can be used like {property}{direction}-{size}

Reply:
As I understand, in some cases one needs to use spacing helpers on top of existing predefined padding/margin defaults. But here I'm surprised that we have to use it that often.

So spacings are not defined as part of components ? Does `v-btn` have default paddings as part of the theme or do I need to explicitly set them every time I define a button ?

Reply:
We could set this globally and override if needed i adjusted this

Reply:
as far as I understand, UIkit should not interfere with existing elements as it's "opt-in", you need to explicitly set the classes. So I don't think it''s a side effect.

You guys have decided to go with "pa-2" for whatever reason, maybe it was visually fitting. If possible it would be good to bring this to the default as soon as possible as I'm afraid that the markup becomes unmaintainable if we start adding such exceptions everywhere.

I'm fine merging this PR first then please address this as next task.